refinery/config/settings/base.py

```diff
@@ -315,7 +315,7 @@ def get_setting(name, settings=local_settings, default=None):
 CELERYD_MAX_TASKS_PER_CHILD = get_setting("CELERYD_MAX_TASKS_PER_CHILD")
 CELERY_ROUTES = {"file_store.tasks.FileImportTask": {"queue": "file_import"}}
 CELERY_ACCEPT_CONTENT = ['pickle']
-CELERYD_TASK_SOFT_TIME_LIMIT = 60  # seconds
+CELERYD_TASK_SOFT_TIME_LIMIT = 180  # seconds
```

Comment:
You've mentioned in the description that the new changes provide a "5x or greater" speed up, so is this really necessary?

Reply:
I suppose not